### PR TITLE
 Added Properties to the Structured Configuration docs

### PR DESCRIPTION
**Pull request by Silcoish**
Added mention of Properties file format alongside JSON, YAML and XML. Added the weather example to the properties section and mentioned that the variable replacement gets encoded and that hirachical replacement isn't possible in properties files.

### CODE REVIEW

File: docs/shared-content/structured-configuration-variables.include.md [hunk 1]
```diff
@@ -375,3 +375,31 @@ One limitation is that if the same prefix is declared more than once in a docume
...
+weatherApiKey = TEST7654321,
+tempImageFolder = D:\\temp\\img,
+logsFolder = C:\\logs
+port = 80,
+debug = false
+```
+
+Note that the `logsFolder` setting remains untouched as there was no variable defined to override the value and that `tempImageFolder` has been encoded with the double `\`. Octopus will encode the variable in the correct encoding for the properties file format. 
+
+Unlike JSON, YAML, and XML, it's not possible to do hirachical replacement in a properties file as properties files are simple key value files.
```
Comment (andrew-moore-octo) [inline docs/shared-content/structured-configuration-variables.include.md:405]:
```suggestion
Unlike JSON, YAML, and XML, it's not possible to do hierachical replacement in a properties file as properties files are simple key value files. 
```

File: docs/shared-content/structured-configuration-variables.include.md [hunk 2]
```diff
@@ -375,3 +375,31 @@ One limitation is that if the same prefix is declared more than once in a docume
 ```
 The namespace 'http://octopus.com' could not be mapped to the 'octopus' prefix, as another namespace 'http://octopus.com/xml' is already mapped to that prefix. XPath selectors using this prefix may not return the expected nodes. You can avoid this by ensuring all namespaces in your document have unique prefixes.
 ```
+
+## Java Properties 
+
+Given this example of a target properties file:
+
+```
+weatherApiUrl = dev.weather.com,
```
Comment (andrew-moore-octo) [inline docs/shared-content/structured-configuration-variables.include.md:384]: The properties file format doesn't have commas at the end of the line. The end of the line itself is the delimiter.

```suggestion
weatherApiUrl = dev.weather.com
```

File: docs/deployment-process/configuration-features/structured-configuration-variables-feature.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Structured configuration variables
-description: The Structured Configuration Variables feature updates your JSON, YAML, and XML configuration files with the values from matching Octopus variables.
+description: The Structured Configuration Variables feature updates your JSON, YAML, XML and Properties configuration files with the values from matching Octopus variables.
```
Comment (wordlee) [inline docs/deployment-process/configuration-features/structured-configuration-variables-feature.md:3]:
```suggestion
description: The Structured Configuration Variables feature updates your JSON, YAML, XML, and Properties configuration files with values from matching Octopus variables.
```

File: docs/deployment-process/configuration-features/xml-configuration-variables-feature.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ This feature can be enabled for package deploy steps.
 Octopus will extract your package and parse your `*.config` files looking for any `appSettings`, `connectionStrings`, and `applicationSettings` elements where the name matches one of your [variables](/docs/projects/variables/index.md).
 
 :::hint
-You can perform simple convention-based replacements in .NET XML configuration files using this feature. We also have a feature tailored to [JSON, YAML, and XML configuration files](/docs/deployment-process/configuration-features/structured-configuration-variables-feature.md).
+You can perform simple convention-based replacements in .NET XML configuration files using this feature. We also have a feature tailored to [JSON, YAML, XML and Properties configuration files](/docs/deployment-process/configuration-features/structured-configuration-variables-feature.md).
```
Comment (wordlee) [inline docs/deployment-process/configuration-features/xml-configuration-variables-feature.md:16]:
```suggestion
You can perform simple convention-based replacements in .NET XML configuration files using this feature. We also have a feature tailored to [JSON, YAML, XML, and Properties configuration files](/docs/deployment-process/configuration-features/structured-configuration-variables-feature.md).
```

File: docs/shared-content/structured-configuration-variables.include.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 :::info
-This Configuration Feature used to be called JSON Configuration Variarbles in versions before 2020.4.0, but has been re-named to Structured Configuration Variables with the added support for YAML and XML replacement.
+This Configuration Feature used to be called JSON Configuration Variarbles in versions before 2020.4.0, but has been re-named to Structured Configuration Variables with the added support for YAML, XML, and Properties configuration file replacements.
```
Comment (wordlee) [inline docs/shared-content/structured-configuration-variables.include.md:2]: Slight tweak for readability:
```suggestion
This Configuration Feature was previously called JSON Configuration Variables. In version **2020.4.0**, we added support for YAML, XML, and Properties configuration file replacements and renamed the feature Structured Configuration Variables.
```

File: docs/shared-content/structured-configuration-variables.include.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
...
 :::
 
-With the **Structured Configuration Variables** feature you can define [variables](/docs/projects/variables/index.md) in Octopus for use in JSON, YAML, and XML configuration files of your applications. This lets you define different values based on the scope of the deployment. Settings are located using a structure-matching syntax, so you can update values nested inside structures such as JSON objects and arrays, YAML mappings and sequences, and XML elements and attributes. XPath is used for XML files, and similar expressions are used for the other formats.
+With the **Structured Configuration Variables** feature you can define [variables](/docs/projects/variables/index.md) in Octopus for use in JSON, YAML, XML and Properties configuration files of your applications. This lets you define different values based on the scope of the deployment. Settings are located using a structure-matching syntax, so you can update values nested inside structures such as JSON objects and arrays, YAML mappings and sequences, and XML elements and attributes. XPath is used for XML files, and similar expressions are used for the other formats.
```
Comment (wordlee) [inline docs/shared-content/structured-configuration-variables.include.md:5]:
```suggestion
With the **Structured Configuration Variables** feature you can define [variables](/docs/projects/variables/index.md) in Octopus for use in JSON, YAML, XML, and Properties configuration files of your applications. This lets you define different values based on the scope of the deployment. Settings are located using a structure-matching syntax, so you can update values nested inside structures such as JSON objects and arrays, YAML mappings and sequences, and XML elements and attributes. XPath is used for XML files, and similar expressions are used for the other formats.
```

File: docs/shared-content/structured-configuration-variables.include.md (hunk 1 above)
Comment (wordlee) [inline docs/shared-content/structured-configuration-variables.include.md:405]:
```suggestion
Unlike JSON, YAML, and XML, it's not possible to do hierarchical replacement in a properties file as properties files are simple key value files. 
```

File: docs/shared-content/structured-configuration-variables.include.md (hunk 2 above)
Comment (Silcoish) [reply docs/shared-content/structured-configuration-variables.include.md]: Thankyou, im really glad you caught this. Bad copy-paste from the json example!

**Pull request merged**